app/controllers/admin_controller.rb

```diff
@@ -12,11 +12,16 @@ def index
   def users
     redirect_to(root_url) && return unless current_user.can_access_admin_panel?
 
-    permitted_params = params.permit(:search, :entries, :page, :orderBy, :sortDirection)
+    permitted_params = params.permit(:search, :entries, :page, :orderBy, :sortDirection, :locked)
 
     # Validate search param
     search = permitted_params[:search]
 
+    # Validate locked param
+    locked = permitted_params[:locked].to_s == 'true' unless permitted_params[:locked].nil?
+    error_message = 'Invalid value for field: "locked", acceptable values are true, false or null'
```

Comment:
I don't think we need to include "null" here, because they should just not even include the `locked` key in `params`, it's only included in the validation array below because when it's not passed, it will be `nil`

Reply:
done here: https://github.com/SaraAlert/SaraAlert/pull/756/commits/f3e0b4cebb6b7c87bd16de586c828d9bc1d8f070